docs/en/connector-v2/source/Hbase.md

```diff
@@ -0,0 +1,91 @@
+# Hbase
+
+> Hbase source connector
+
+## Description
+
+Read data from Apache Hbase.
+
+## Key features
+
+- [x] [batch](../../concept/connector-v2-features.md)
+- [ ] [stream](../../concept/connector-v2-features.md)
+- [ ] [exactly-once](../../concept/connector-v2-features.md)
+- [ ] [schema projection](../../concept/connector-v2-features.md)
+- [x] [parallelism](../../concept/connector-v2-features.md)
+- [ ] [support user-defined split](../../concept/connector-v2-features.md)
+
+## Options
+
+|        name        |  type  | required | default value |
+|--------------------|--------|----------|---------------|
+| zookeeper_quorum   | string | yes      | -             |
+| table              | string | yes      | -             |
+| query_columns      | list   | yes      | -             |
```

Comment:
From this option we can implement `schema projection` ? If true, You need update here 
```
[x] [schema projection](../../concept/connector-v2-features.md)
```
and the `HbaseSource` need implement `SupportColumnProjection` interface.

`HbaseSource` need implement `SupportParallelism` too, because it support `parallelism `.